src/components/VaultSummary.tsx

```diff
@@ -10,7 +10,8 @@ import { calculateCurrentDebt } from '@agoric/inter-protocol/src/interest-math';
 import {
   ceilMultiplyBy,
   makeRatioFromAmounts,
-} from '@agoric/zoe/src/contractSupport';
+  multiplyBy,
+} from '@agoric/zoe/src/contractSupport/ratio';
```

Comment:
these are all in `@agoric/ertp` now. since this app is going away I won't push for it